readme.md

```diff
@@ -12,6 +12,8 @@ You can view documentation on how to get started [here](https://loki-project.git
 
 [![Build Status](https://drone.lokinet.dev/api/badges/loki-project/loki-network/status.svg?ref=refs/heads/master)](https://drone.lokinet.dev/loki-project/loki-network)
 
+You can find Bleeding edge builds [here](https://builds.lokinet.dev) (these builds may eat your first born).
```

Comment:
Amusing, I suppose, but...probably not the *best* thing to have in the readme.

Reply:
that was added in a commit to force ci to rebuild

Reply:
removed